integrations/couchdb/documentation.yaml

```diff
@@ -0,0 +1,83 @@
+exporter_type: sidecar
+app_name_short: CouchDB
+app_name: {{app_name_short}}
+app_site_name: CouchDB
+app_site_url: https://couchdb.apache.org/
+exporter_name: the CouchDB exporter
+exporter_pkg_name: couchdb-prometheus-exporter
+exporter_repo_url: https://github.com/gesellix/couchdb-prometheus-exporter
+dashboard_available: true
+minimum_exporter_version: v30.4.3
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+config_mods: |
+  apiVersion: apps/v1
+  kind: StatefulSet
+  metadata:
+    name: couchdb
+  spec:
+    serviceName: couchdb
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: couchdb
+    template:
+      metadata:
+        labels:
+  +       app.kubernetes.io/name: couchdb
+      spec:
+        containers:
+        - name: couchdb
+          image: couchdb:3.2.2
+          env:
+            - name: COUCHDB_USER
+              value: "otelu"
+            - name: COUCHDB_PASSWORD
+              value: "otelp"
+            - name: NODENAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: ERL_FLAGS
+              value: "-setcookie otel"
+          ports:
+          - containerPort: 5984
+  +     - name: couchdb-exporter
+  +       image: gesellix/couchdb-prometheus-exporter:v30.4.3
+  +       args:
+  +       - --couchdb.uri=http://localhost:5984
+  +       - --logtostderr
+  +       env:
+  +         - name: COUCHDB_USERNAME
+  +           value: "otelu"
+  +         - name: COUCHDB_PASSWORD
+  +           value: "otelp"
+  +       ports:
+  +       - containerPort: 9984
+  +         name: prometheus
+additional_install_info: |
+  These instructions assume you already have a working {{app_name_short}}
+  installation and want to modify it to include an exporter.
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: couchdb
+    labels:
+      app.kubernetes.io/name: couchdb
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: prometheus
+      scheme: http
+      interval: 60s
+      path: /metrics
+      params:
+        format: 
+          - prometheus
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: couchdb
+additional_prereq_info: |
+  Update the exporter's `COUCHDB_USERNAME` and `COUCHDB_PASSWORD` environment variables
+  to that of your monitoring user.
```

Comment:
I looked at the template again...

```
<h2>Prerequisites</h2>

<p>To collect metrics from {{app_name_short}} by using Google Managed Service for Prometheus and managed collection, your deployment must meet the following requirements:
- ...
- ...

{{additional_prereq_info}}

<h2>Install the {{app_name_short}} exporter</h2>

<p>We recommend that you install the {{app_name_short}} exporter, {{exporter_repo_link}},
as a sidecar to your {{app_name_short}} workload. For information about using sidecars, see
<a>Extended applications on Kubernetes with multi-container pods</a>.</p>

<p>To install {{exporter_literal}} as a sidecar to {{app_name_short}}, modify your {{app_name_short}} configuration as shown in the following example:</p>

<pre>
{{config_mods}}
</pre>

<p>You must add any lines preceded by the <code>+</code> symbol to your
configuration.</p>

{{additional_install_info}}
```

{{additional_prereq_info}} precedes the entire section about exporter. Given this, I feel that this sentence fits better under `additional_install_info`, because it relates to setting up the exporter sidecar correctly, and should follow the {{config_mod}} code block.